probabilistic.go

```diff
@@ -165,6 +169,79 @@ func (c cmdable) BFExists(ctx context.Context, key string, element interface{})
 	return cmd
 }
 
+func (c cmdable) BFLoadChunk(ctx context.Context, key string, iterator int64, data interface{}) *StatusCmd {
+	args := []interface{}{"bf.loadchunk", key, iterator, data}
+	cmd := NewStatusCmd(ctx, args...)
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+func (c cmdable) BFScanDump(ctx context.Context, key string, iterator int64) *ScanDumpCmd {
```

Comment:
WDYT let's start doing what we do in other clients - add some boilerplate to the function explaining what it is and does - link to the redis.io docs?